lpc55_sign/src/signed_image.rs

```diff
@@ -10,15 +10,17 @@ use der::Encode as _;
 use lpc55_areas::*;
 use packed_struct::prelude::*;
 use rsa::{traits::PublicKeyParts, RsaPrivateKey};
-use serde::Deserialize;
+use serde::{Deserialize, Serialize};
 use sha2::{Digest, Sha256};
 use x509_cert::Certificate;
 
-#[derive(Clone, Debug, Deserialize)]
+/// Struct defining the TOML format for --cert-cfg, which bundles up flags that
```

Comment:
```suggestion
/// Struct defining the TOML format for `--cert-cfg`, which bundles up flags that
```

Reply:
Heh, I stopped including the backticks because Clap just prints them to the shell and it looks weird. But I can backtick it.